### ISSUE TITLE
Broken on Node 10.0.0

### ISSUE THREAD

**Issue by Extarys** [masumsoft/express-cassandra#160]
```
models.instance.User.findOne({ userId: "14f12dac-2ba6-4e15-81e6-4c5692278af0" }, function(err, user) {
if(err) {
	        	console.log(err);
			    }

})
```

Log:
```
Unhandled rejection TypeError: frame.getFileName is not a function
    at isInsideNodeModules (internal/util.js:360:28)
    at showFlaggedDeprecation (buffer.js:149:8)
    at new Buffer (buffer.js:174:3)
    at Array.<anonymous> (/media/srv/NodesProjects/NJS-/Api/node_modules/source-map-support/source-map-support.js:149:21)
    at /media/srv/NodesProjects/NJS-/Api/node_modules/source-map-support/source-map-support.js:53:24
    at mapSourcePosition (/media/srv/NodesProjects/NJS-/Api/node_modules/source-map-support/source-map-support.js:171:21)
    at wrapCallSite (/media/srv/NodesProjects/NJS-/Api/node_modules/source-map-support/source-map-support.js:343:20)
    at /media/srv/NodesProjects/NJS-/Api/node_modules/source-map-support/source-map-support.js:378:26
    at Array.map (<anonymous>)
    at Function.prepareStackTrace (/media/srv/NodesProjects/NJS-/Api/node_modules/source-map-support/source-map-support.js:377:24)
    at formatError (util.js:758:16)
    at formatValue (util.js:609:17)
    at inspect (util.js:336:10)
    at Object.formatWithOptions (util.js:190:12)
    at Console.(anonymous function) (console.js:186:15)
    at Console.log (console.js:197:31)

```

I wanted to report the issue since I'm checking compatibility using nvm.

**Comment by masumsoft** [comment]
Thanks for reporting, we don't yet have official support for node 10, but we'll have a look at it.

**Comment by masumsoft** [comment]
This module works fine with Node 10 and actually passes [all existing tests](https://travis-ci.org/masumsoft/express-cassandra). The issue you are reporting is a bug of nodejs 10.0.0 which was fixed in nodejs version 10.1.0. You should probably use nodejs version >= 10.1.0
